cli/azd/pkg/apphost/generate.go

```diff
@@ -1570,7 +1409,6 @@ func (b *infraGenerator) buildArgsBlock(args []string, manifestCtx *genContainer
 		// This logic is similar to what we do in buildEnvBlock to detect when we need to take values and treat them as ACA
 		// secrets.
 		if strings.Contains(arg, ".connectionString}") ||
-			strings.Contains(resolvedArg, "{{ connectionString") ||
```

Comment:
container.v0 has `connectionString` (https://github.com/dotnet/aspire/blob/main/src/Schema/aspire-8.0.json#L64)

Reply:
This case should be handled today.  This is removing the support for the `connectionString` function which was supported in our container app templates and only supported a subset of resources (the old `azure.*` onces, as well as postgres and redis for the ACA dev services) and none of these types of resources are supported anymore, so we will never emit calls to this helper.  I have also deleted the helper method as part of this PR.